packages/react-native-reanimated/package.json

```diff
@@ -96,7 +96,8 @@
   "peerDependencies": {
     "@babel/core": "^7.0.0-0",
     "react": "*",
-    "react-native": "*"
+    "react-native": "*",
+    "react-native-worklets": "0.0.1"
```

Comment:
`0.0.1` is okay for now, but what's the plan for the future? Note that `peerDependencies` are not respected with `npm i --force`. Maybe let's just use `*` here and add some more sophisticated checks in Reanimated?